romana-install/group_vars/vagrant

```diff
@@ -1,7 +1,27 @@
 ram_size:
   controller:
     devstack: 4096
-    kubernetes: 1024
+    kubernetes: 2048
   compute:
     devstack: 1024
```

Comment:
It's just strange to see that a k8s compute host would need more memory than an OpenStack compute host.